core/object/object.h

```diff
@@ -49,7 +49,7 @@ class TypedArray;
 
 enum PropertyHint {
 	PROPERTY_HINT_NONE, ///< no hint provided.
-	PROPERTY_HINT_RANGE, ///< hint_text = "min,max[,step][,or_greater][,or_less][,hide_slider][,radians_as_degrees][,degrees][,exp][,suffix:<keyword>] range.
+	PROPERTY_HINT_RANGE, ///< hint_text = "min,max[,step][,or_greater][,or_less][,hide_slider][,radians_as_degrees][,degrees][,exp][,display_unsigned][,suffix:<keyword>] range.
```

Comment:
I wonder if we should go beyond just `display_unsigned` (which is only about display) to something more like `treat_as_unsigned` or similar name (which is more about how to interpret the data)?

There's a couple of places in Godot's API where we treat the value of an integer as unsigned, but we don't have a way to formally mark it as such. This has come up in the context of GDExtension a number of times. We've decided to assume that bitfields are unsigned, which covers many of these cases. But having a property hint like this would help with any remaining ones.

cc @Bromeon

Reply:
That's a good point, would be interested in looking at it, could name it that way here and leave things open for future expansion